src/texture.jl

```diff
@@ -104,29 +104,47 @@ CuTextureArray(A::AbstractArray{T,N}) where {T,N} = CuTextureArray{T,N}(A)
 
 ## memory operations
 
-function Base.copyto!(dst::CuTextureArray{T,1}, src::Union{Array{T,1}, CuArray{T,1}}) where {T}
+function Base.copyto!(dst::CuTextureArray{T,1}, src::Union{Array{T,1}, CuArray{T,1}, CuTextureArray{T,3}}) where {T}
```

Comment:
Why this change?

Reply:
To allow copying from one texture array to another? In the `unsafe_copy3d!`, `ArrayMemory` is supposedly supported (https://github.com/JuliaGPU/CUDA.jl/blob/b51c6228621cc37bc3fb40d6b3467487c8e02063/lib/cudadrv/memory.jl#L558) but I think is unreachable from the `copyto!` without this change

Reply:
Oh I see, I have a typo here - 3 should be 1, now fixed